less/ovh-angular-sidebar-menu.dist.less

```diff
@@ -1,3 +1,4 @@
-@import "ovh-angular-sidebar-menu.less";
+@import "../bower_components/ovh-ui-kit/packages/oui-responsive/_variables";
```

Comment:
We should find a different way of doing this instead of continuous to deal with Bower path.

Reply:
Since it's a bower dependency, do you have another way to do it ?
It will be the same for node_modules